api/info/info.go

```diff
@@ -50,8 +51,28 @@ func (h *Handler) Handle(w http.ResponseWriter, r *http.Request) error {
 	if err != nil {
 		return err
 	}
+
+	var profile *config.SigningProfile
+	policy := h.sign.Policy()
+	if policy != nil && policy.Profiles != nil && req.Profile != "" {
+		profile = policy.Profiles[req.Profile]
+	}
+	if profile == nil && policy != nil {
+		profile = policy.Default
+	}
+	var usage []string
+	var expiry string
+	if profile == nil {
```

Comment:
per @kisom, if the profile is nil, response with error. So ignore what I suggested.